### DISCUSSION TITLE
Attention layers in the Unet Model

### DISCUSSION THREAD

**Discussion by lobstercare** [qubvel/segmentation_models.pytorch#709] [General]
Hello Everyone, 
So I'm doing an Image segmentation with the Unet Model and I've drawned the net with torchvision and found out that there's attention layers, can someone tell me why ? , because I didn't know that there were Attention layers in the Unet model, unless if it's an Attention Unet ! 
<img width="254" alt="interpo" src="https://user-images.githubusercontent.com/47992103/211782963-02792e09-b87d-4416-acd3-2ab661ac6402.PNG">

Thanks everyone !

**Comment by cagey-squirrel** [comment]
This attention layer is just the identity function by default so you can ignore it.
By setting ```attention='scse'``` in your model you can change this layer to be scSE attention layer and this is done only in decoder.
